script-opts/contact_sheet.conf

```diff
@@ -1,4 +1,8 @@
-# options that are common to both scripts are documented in playlist_view.conf
+# mpv-gallery-view | https://github.com/occivink/mpv-gallery-view
+# This is the settings file for scripts/contact-sheet.lua
+# File placement: script-opts/contact_sheet.conf
+# Additional settings can be configure in script-opts/playlist_view.conf
```

Comment:
Settings from `playlist_view.conf` are not loaded by the contact-sheet script. The previous comment simply indicated that settings that are common to both scripts (for example, `gallery_position`) are only documented in the `script-opts/playlist_view.conf` file inside the repo.

Reply:
Sorry, my mistake. I fixed it.